packages-ts/terra-sdk/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import { WebSocketClient } from '@terra-money/terra.js';
+import { TxLog, Int } from '@terra-money/terra.js';
+
+export interface Round {
+    answer: Int;
+    round: number;
```

Comment:
To be consistent and avoid `round.round` situations:

```suggestion
    roundId: number;
```